MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -3901,6 +3916,19 @@ public boolean isTech() {
         return isTechMek() || isTechAero() || isTechMechanic() || isTechBA();
     }
 
+    /**
+     * Checks if the person is a tech, includes mektek, mechanic, aerotek, BAtek and the non-cannon "large vessel tek"
+     * @return true if the person is a tech
+     */
+    public boolean isTechExpanded() {
```

Comment:
Are there other functions than maintaining a mothballed thing, or a more descriptive name for this group that could be used for a method name?

Reply:
its exclusivelly used by the function `pickTechForMothballOrActivation`

I can come up with a better name, but `expanded` does cover pretty well the intent, which brings `techs` and an expansion on the variety.